### PR TITLE
Like count and display

### PR DESCRIPTION
**Pull request by sentayhu19**
**in this pull req**

- show Like 
- Like Count

### CODE REVIEW

File: src/modules/getvshow.js [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+  renderShows();
+};
+const getlikes = (async () => {
+  const response = await fetch(url, { method: 'GET' });
+  return response.json();
```
Comment (deyemiobaa) [inline src/modules/getvshow.js:24]: The default method for fetch is GET, so there's no need to specify it again in the function

File: src/modules/render.js [hunk 2]
```diff
@@ -18,18 +26,26 @@ const clickEvents = () => {
...
-    <i class="fa-solid fa-heart">
+    <i class="fa-solid fa-heart like-btn">
     </i>
-    <p></p>
+    <p> ${likeCount}</p>
     <i class="fa-solid fa-comment"></i>
     </div>
     </li>
     `;
+    likeCount = 0;
```
Comment (deyemiobaa) [inline src/modules/render.js:48]: I noticed that you are reassigning a value to `likeCount` here. I think that isn't needed

**Review by deyemiobaa — APPROVED** [review]
Hello @sentayhu19. You've done a great job and the functionalities are working fine. 
I stated a few change requests in the comment below, they don't affect the functionalities it's just to keep the code clean. 
Kindly check them out

File: src/modules/render.js (hunk 2 above)
Comment (sentayhu19) [reply src/modules/render.js:48]: that's to reset the value back to 0

**Pull request merged**